pkg/controller/podreadiness/pod_readiness_controller.go

```diff
@@ -129,12 +132,23 @@ func (r *ReconcilePodReadiness) Reconcile(_ context.Context, request reconcile.R
 			return nil
 		}
 
-		pod.Status.Conditions = append(pod.Status.Conditions, v1.PodCondition{
-			Type:               appspub.KruisePodReadyConditionType,
-			Status:             v1.ConditionTrue,
-			LastTransitionTime: metav1.Now(),
-		})
-		return r.Status().Update(context.TODO(), pod)
+		// patch pod condition
+		status := v1.PodStatus{
```

Comment:
It is not necessary to use the RetryOnConflict method when using the patch method.

Reply:
Is it ok now?